lib/mixins/helpers.js

```diff
@@ -152,15 +139,14 @@ var helpers = {
       'slick-cloned': slickCloned
     });
   },
-  getListStyle: function () {
-    var style = {};
+  adaptHeight: function () {
     if (this.props.adaptiveHeight) {
       var selector = '[data-index="' + this.state.currentSlide +'"]';
       if (this.refs.list) {
-        style.height = this.refs.list.getDOMNode().querySelector(selector).offsetHeight;
+        var slickList = this.refs.list.getDOMNode();
+        slickList.style.height = slickList.querySelector(selector).offsetHeight + 'px';
```

Comment:
Yeah. The warning said getDOMNode operation should be done inside `componentDidMount` or `componentDidUpdate` methods only.